ops/Makefile

```diff
@@ -59,55 +63,103 @@ promote-%: .be-logged-in .valid-env-%
+	if [[ "$*" == "prod" ]] ; then \
+	  until [[ "$(RELEASE_TAG)"  == "`$(CURL) $(PROD_SLOT_INFO_ENDPOINT) | jq -r .deploy.tag`" ]]; do sleep 5; done ; \
+	else \
+	  until [[ "$(RELEASE_TAG)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .deploy.tag`" ]]; do sleep 5; done ; \
+	fi;
 	@echo "Finished waiting at `date`"
 
 wait-for-%-slot-readiness: .valid-env-% # could use curl's built-in retry/timeout if we wanted to
 	@echo "Waiting for a successful readiness probe in $*/$(SOURCE_SLOT)"
-	until $(SLOT_READY); do sleep 5; done;
+	if [[ "$*" == "prod" ]] ; then \
+	  until $(PROD_SLOT_READY); do sleep 5; done; \
+	else \
+	  until $(SLOT_READY); do sleep 5; done; \
+	fi;
 
 wait-for-%-readiness: .valid-env-% # could use curl's built-in retry/timeout if we wanted to
 	@echo "Waiting for a successful readiness probe in $*"
-	until $(APP_READY); do sleep 5; done;
+	if [[ "$*" == "prod" ]] ; then \
+	  until $(PROD_APP_READY); do sleep 5; done; \
+	else \
+	  until $(APP_READY); do sleep 5; done; \
+	fi;
 
 check-%-slot-commit: .valid-env-%
 	@echo Checking that $(DEPLOYED_COMMIT) is the current deployed commit for $*/$(SOURCE_SLOT)
-	[[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]]
+	if [[ "$*" == "prod" ]] ; then \
+	  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(PROD_SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; \
+	else \
+	  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; \
+	fi;
 
 check-%-commit: .valid-env-%
 	@echo Checking that $(DEPLOYED_COMMIT) is the current deployed commit for $*
-	[[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(INFO_ENDPOINT) | jq -r .git.commit.id`" ]]
+	if [[ "$*" == "prod" ]] ; then \
+	  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(PROD_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; \
+	else \
+	  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; \
+	fi;
 
 check-%-slot-release: .valid-env-%
 	@echo Checking that $(RELEASE_TAG) is the current release on $*/$(SOURCE_SLOT)
-	[[ "$(RELEASE_TAG)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .deploy.tag`" ]]
+	if [[ "$*" == "prod" ]] ; then \
+	  [[ "$(RELEASE_TAG)"  == "`$(CURL) $(PROD_SLOT_INFO_ENDPOINT) | jq -r .deploy.tag`" ]] ; \
+	else \
+	  [[ "$(RELEASE_TAG)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .deploy.tag`" ]] ; \
+	fi;
 
 check-%-release: .valid-env-%
 	@echo Checking that $(RELEASE_TAG) is the current release on $*
-	[[ "$(RELEASE_TAG)"  == "`$(CURL) $(INFO_ENDPOINT) | jq -r .deploy.tag`" ]]
+	if [[ "$*" == "prod" ]] ; then \
+	  [[ "$(RELEASE_TAG)"  == "`$(CURL) $(PROD_INFO_ENDPOINT) | jq -r .deploy.tag`" ]] ; \
+	else \
+	  [[ "$(RELEASE_TAG)"  == "`$(CURL) $(INFO_ENDPOINT) | jq -r .deploy.tag`" ]] ; \
+	fi;
 
 check-%-slot-readiness: .valid-env-%
 	@echo Checking the readiness probe for $*/$(SOURCE_SLOT)
-	$(SLOT_READY)
+	@if [[ "$*" == "prod" ]] ; then \
```

Comment:
Curious why we want to suppress the echo of `if` only for this conditional and the one below? (I don't interact with Makefiles very often, so apologies if this is a silly question!)

Reply:
I appreciate you calling this out! That was my error on copy-paste. Fixed!